lib/facebook_ads/configuration.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+module FacebookAds
+  class Configuration
+    def initialize
+      @config = {}
+    end
+
+    def configured?
```

Comment:
I will not call it `present?` because of Rails inhibiting this method name and thereby generating expectations.

`valid?` it is!